pages/saved.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Image from 'next/image'
+import { FaArrowRight } from "react-icons/fa";
+import Link from 'next/link';
+
+const Saved = () => {
+  return (
+    <>
+      <div className="p-8">
```

Comment:
What do you think of shrinking the padding for the mobile version?

```suggestion
      <div className="p-2 sm:p-8">
```